.github/workflows/quickstart.yaml

```diff
@@ -1,10 +1,9 @@
 name: Quickstart
 
 on:
-  push:
-    branches: main
-  pull_request:
-    branches: main
+  schedule:
```

Comment:
If we do this we will be running this every day. I'm not sure this is something we want to do. But we can comment on how to do it.

Reply:
Sure, we can change it to weekly or monthly.